app/views/deals/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<section class="container">
+    <%= form_with url: group_deals_path, method: :post do |f| %>
+      <div class="field my-3">
+        <%= f.text_field :name, autofocus: true, class: "form-control", placeholder: "name" %>
+      </div>
+  
+      <div class="field my-3">
+        <%= f.number_field :amount, autofocus: true, class: "form-control", placeholder: "amount" %>
+      </div>
+  
+      <div class="field my-3">
+        <%= f.select :groups, options_from_collection_for_select(@groups, 'id', 'name'), {}, { class: "form-control" }%>
+      </div>
+  
+      <div class="text-center">
+        <%= f.submit "Add Deal",  class: "btn btn-primary" %>
+      </div>
+    <% end %>
+  </section>
```

Comment:
- [ ] From the requirement I am on the new transaction page, there should be a "<" button on the navbar that takes me to the transactions index page when I click it, but you do not have the button. Kindly add this arrow button to the navbar and ensure it takes users to the transaction index page.

  ![image](https://github.com/ahmedeid6842/Budget_App/assets/36542986/1ab7a0a9-ed7d-43fa-86ed-6116d9a37233)

Reply:
Fixed